schema/project-level/project-schema.json

```
@@ -657,10 +657,10 @@
       "properties": {
         "scheme": {
           "title": "Scheme",
-          "description": "The scheme or codelist from which the classification code is taken.",
-          "type": [
-            "string"
-          ],
+          "description": "The additional classification scheme, using the open [additionalClassificationsScheme](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/reference/codelists/#additionalclassificationsscheme) codelist.",
```

Keep the original sentence, which better explains what a scheme is, rather than simply restating the title of the field.

```suggestion
          "description": "The scheme or codelist from which the classification code is taken, using the open [additionalClassificationsScheme](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/reference/codelists/#additionalclassificationsscheme) codelist.",
```